components/CreateSetupLink.tsx

```diff
@@ -81,10 +82,7 @@ const CreateSetupLink = (props: { service: 'sso' | 'dsync' }) => {
 
   return (
     <>
-      <Link href='' onClick={() => router.back()} className='btn-outline btn items-center space-x-2'>
-        <ArrowLeftIcon aria-hidden className='h-4 w-4' />
-        <span>Back</span>
-      </Link>
+      <ButtonBack onClick={() => router.back()} />
```

Comment:
I think semantically a link would be a better choice here as we are doing navigation (similar to what we have in Add/Edit). We could support onClick with `LinkBack` component.

Reply:
Do Links support onClick?